rules/linux/lnx_local_group.yml

```diff
@@ -0,0 +1,27 @@
+title: Local Groups
+id: 0d68a143-a178-4aa6-b3f5-e441aceca810
+status: stable
+description: Detects group enumeration
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://attack.mitre.org/techniques/T1069/001/
```

Comment:
could you please use [this](https://github.com/Neo23x0/sigma/pull/1044#discussion_r502045513) recommendation here as well?